### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by inson1**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ body:
     required: true
 - type: dropdown
   attributes:
-    label: What arch are you using?
+    label: What architecture are you using?
```
Comment (ArjixWasTaken) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: if you are going to update this, then specify "CPU architecture" instead.

**Review by ArjixWasTaken — CHANGES_REQUESTED** [review]
just one thing from me

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (inson1) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: done

**Review by ArjixWasTaken — APPROVED** [review]

**Comment by inson1** [comment]
@ArjixWasTaken btw what do you think of enhancement issue template?

**Pull request merged**